packages/endpoints-organizations-registrar/test/organization-controller.test.js

```diff
@@ -1561,6 +1561,7 @@ describe('Organization Registrar Test Suite', () => {
                 id: '#credentialagent-999',
                 type: ServiceTypes.HolderAppProviderType,
                 serviceEndpoint: 'https://agent.samplevendor.com',
+                supportedExchangeProtocols: ['VN_API', 'foo'],
```

Comment:
@sloops77 any idea why this `foo` value is being allowed in the response?

The schema specifies:

```
"supportedExchangeProtocols": {
      "type": "array",
      "items": {
        "type": "string",
        "enum": [
          "VN_API",
          "OPENID4VC"
        ]
      },
      "minItems": 1
    }
```

And I can't figure out why the `enum` is not being enforced. It seems generally the `supportedExchangeProtocols` is seen, but maybe not, a bit confused. See anything obvious?

Reply:
@nassan no i have no idea why that wouldn't  work

Reply:
@sloops77 I didn't figure out why, but valid tests pass, so I guess we can not leave for now